docs/intro/tutorial02.md

```diff
@@ -0,0 +1,117 @@
+# Writing your first app, part 2
+
+This tutorial begins where [Tutorial 1](tutorial01.md) left off. We’ll walk through the process to set up your personal
+registry and publish your application to that registry.
+
+For this tutorial, we will be creating a registry hosted on Microsoft Azure, but there are other cloud providers that
+provide their own solutions, and you can [run one on your own infrastructure](https://github.com/docker/distribution),
+too!
+
+## What is a registry, and what is wasm-to-oci?
+
+An OCI ([Open Container Initiative](https://www.opencontainers.org/)) registry is a popular hosting service. A registry
```

Comment:
Calling OCI registry a service is a little misleading. I might consider rewording it to something about how it is a standard specification for storing container images

Reply:
Good call